### PR TITLE
Add go 1.12.7

### PR DESCRIPTION
**Pull request by dgassaway**

### CODE REVIEW

File: build-support/bin/go/mac/10.10/1.12.7/build.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```
Comment (stuhood) [inline build-support/bin/go/mac/10.10/1.12.7/build.sh]: Each of the OSX version directories can probably be symlinks to just one copy of this script for this version: see c803d9f4b3dc9d1bc8e9d662a7795a2c511f8a99 for an example.

Comment (dgassaway) [reply build-support/bin/go/mac/10.10/1.12.7/build.sh]: Thanks - changed that

Comment (dgassaway) [reply build-support/bin/go/mac/10.10/1.12.7/build.sh]: there we go!

**Comment by stuhood** [comment]
```$ ./sync-s3.sh
upload: bin/go/linux/x86_64/1.12.7/build.sh to s3://binaries.pantsbuild.org/bin/go/linux/x86_64/1.12.7/build.sh
upload: bin/go/mac/10.10/1.12.7/build.sh to s3://binaries.pantsbuild.org/bin/go/mac/10.10/1.12.7/build.sh
upload: bin/go/linux/x86_64/1.12.7/go.tar.gz to s3://binaries.pantsbuild.org/bin/go/linux/x86_64/1.12.7/go.tar.gz
upload: bin/go/mac/10.11/1.12.7/build.sh to s3://binaries.pantsbuild.org/bin/go/mac/10.11/1.12.7/build.sh
upload: bin/go/mac/10.12/1.12.7/build.sh to s3://binaries.pantsbuild.org/bin/go/mac/10.12/1.12.7/build.sh
upload: bin/go/mac/10.10/1.12.7/go.tar.gz to s3://binaries.pantsbuild.org/bin/go/mac/10.10/1.12.7/go.tar.gz
upload: bin/go/mac/10.11/1.12.7/go.tar.gz to s3://binaries.pantsbuild.org/bin/go/mac/10.11/1.12.7/go.tar.gz
upload: bin/go/mac/10.13/1.12.7/build.sh to s3://binaries.pantsbuild.org/bin/go/mac/10.13/1.12.7/build.sh
upload: bin/go/mac/10.14/1.12.7/build.sh to s3://binaries.pantsbuild.org/bin/go/mac/10.14/1.12.7/build.sh
upload: bin/go/mac/10.12/1.12.7/go.tar.gz to s3://binaries.pantsbuild.org/bin/go/mac/10.12/1.12.7/go.tar.gz
upload: bin/go/mac/10.14/1.12.7/go.tar.gz to s3://binaries.pantsbuild.org/bin/go/mac/10.14/1.12.7/go.tar.gz
upload: bin/go/mac/10.13/1.12.7/go.tar.gz to s3://binaries.pantsbuild.org/bin/go/mac/10.13/1.12.7/go.tar.gz```

**Pull request merged**